archinstall/lib/configuration.py

```diff
@@ -242,3 +243,63 @@ def preview(item: MenuItem) -> str | None:
 			config_output.save_user_creds(dest_path, password=enc_password)
 		case 'all':
 			config_output.save(dest_path, creds=True, password=enc_password)
+
+
+def has_saved_config() -> bool:
```

Comment:
Can we move these into the `ConfigurationOutput` class and Maybe rename that thing to `ConfigurationHandler` it will make future refactoring much easier